Pod/Classes/SEGAdobeIntegration.m

```diff
@@ -460,6 +477,41 @@ - (void)trackHeartbeatEvents:(SEGTrackPayload *)payload
         SEGLog(@"[ADBMediaHeartbeat trackEvent:ADBMediaHeartbeatEventBufferStart mediaObject:%@ data:%@]", self.mediaObject, payload.properties);
         return;
     }
+
+    // Video Ad Break Started/Completed are not spec'd. For now, will document for Adobe and
+    // write a proposal to add this to the Video Spec
+    if ([payload.event isEqualToString:@"Video Ad Break Started"]) {
+        self.mediaObject = [self createMediaObject:payload.properties andEventType:@"Ad Break"];
+        [self.ADBMediaHeartbeat trackEvent:ADBMediaHeartbeatEventAdBreakStart mediaObject:self.mediaObject data:nil];
+        SEGLog(@"[ADBMediaHeartbeat trackEvent:ADBMediaHeartbeatEventAdBreakStart mediaObject:%@ data:nil]", self.mediaObject);
+        return;
+    }
+
+    if ([payload.event isEqualToString:@"Video Ad Break Completed"]) {
+        [self.ADBMediaHeartbeat trackEvent:ADBMediaHeartbeatEventAdBreakComplete mediaObject:nil data:nil];
```

Comment:
Is the media object nil here? There should be a media object for an ad break completed event. I also did decide to pass non-standard ad properties to an ad break started event. Sorry if I'm not reading this right!